.circleci/config.yml

```diff
@@ -157,6 +158,16 @@ jobs:
                 unzip habitat-test-scenes.zip
                 rm habitat-test-scenes.zip
               fi
+      - run:
+          name: Download coda scene
+          command: |
+            if [ ! -f ./habitat-sim/data/scene_datasets/coda/coda.glb ]
+            then
+              cd habitat-sim
+              wget --no-check-certificate 'https://docs.google.com/uc?export=download&id=1Pc-J6pZzXEd8RSeLM94t3iwO8q_RQ853' -O coda.zip
```

Comment:
OKay, this could break things down the line for sure... We should fix this and properly host it.

Reply:
Yes. Can you help me upload this scene to a  Fb's s3 bucket?

Reply:
Yeah sure. Also you should remove __MAC_OSX subfiles from the zip. No reason to include them.

Reply:
Great! I have removed the subfiles from the latest version of the zip. It's here: https://drive.google.com/file/d/1Pc-J6pZzXEd8RSeLM94t3iwO8q_RQ853/view?usp=sharing